### PR TITLE
fix: client capabilities changes in v7 - WPB-12024

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-12024" title="WPB-12024" target="_blank">TaskWPB-12024</a>  [iOS] Adopt API v7
  </td></table>
  <br />
 

### Issue

This PR adds the change to `Client.capabilities` introduced with v7.

---

### Checklist

- [ ] Title contains a reference JIRA issue number like `[WPB-XXX]`.
- [ ] Description is filled and free of optional paragraphs.
- [ ] Adds/updates automated tests.

---

### UI accessibility checklist

_If your PR includes UI changes, please utilize this checklist:_
- [ ] Make sure you use the API for UI elements that support large fonts.
- [ ] All colors are taken from WireDesign.ColorTheme or constructed using WireDesign.BaseColorPalette.
- [ ] New UI elements have Accessibility strings for VoiceOver.

### CODE REVIEW

File: WireAPI/Tests/WireAPITests/APIs/UserClientsAPI/UserClientsAPITests.swift [hunk 1]
```diff
@@ -85,7 +85,7 @@ final class UserClientsAPITests: XCTestCase {
             for version in testedVersions {
                 // Given
                 let apiService = MockAPIServiceProtocol.withResponses([
-                    (.ok, "GetClientsSuccessResponseV0")
+                    (.ok, "GetUserClientsSuccessResponseV0")
```
Comment (caldrian) [inline WireAPI/Tests/WireAPITests/APIs/UserClientsAPI/UserClientsAPITests.swift]: Not directly related, but Jullian suggested to fix the file name.

File: WireAPI/Sources/WireAPI/APIs/UserClientsAPI/UserClientsAPIV7.swift [hunk 2]
```diff
@@ -16,6 +16,91 @@
...
+    let id: String
+    let type: UserClientType
+    let activationDate: UTCTimeMillis
+    let label: String?
+    let model: String?
+    let deviceClass: DeviceClass?
+    let lastActiveDate: UTCTime?
+    let mlsPublicKeys: MLSPublicKeys?
+    let cookie: String?
+    let capabilities: [UserClientCapability]?
```
Comment (jullianm) [inline WireAPI/Sources/WireAPI/APIs/UserClientsAPI/UserClientsAPIV7.swift]: @caldrian  I would add a small comment to specify what changed in v7 (removal of the object container)

**Review by jullianm — APPROVED** [review]
looks good, just left a minor comment

File: WireAPI/Tests/WireAPITests/APIs/UserClientsAPI/UserClientsAPITests.swift (hunk 1 above)
Comment (jullianm) [reply WireAPI/Tests/WireAPITests/APIs/UserClientsAPI/UserClientsAPITests.swift]: maybe let's call it `GetOtherUserClientsSuccessResponseV0` to be more precise

File: WireAPI/Sources/WireAPI/APIs/UserClientsAPI/UserClientsAPIV7.swift (hunk 2 above)
Comment (caldrian) [reply WireAPI/Sources/WireAPI/APIs/UserClientsAPI/UserClientsAPIV7.swift]: fixed, thank you for the hint!

File: WireAPI/Tests/WireAPITests/APIs/UserClientsAPI/UserClientsAPITests.swift (hunk 1 above)
Comment (caldrian) [reply WireAPI/Tests/WireAPITests/APIs/UserClientsAPI/UserClientsAPITests.swift]: done!

**Pull request merged**